tools/releaser/scripts/breaking-changes.sh

```diff
@@ -3,24 +3,21 @@ set -eu
 GOPATH=$(go env GOPATH)
 
 # Inputs:
-# GIT_BASE_REF - The base REF of the git repository (git rev-parse origin/main)
-# Usually "${{ github.event.pull_request.base.sha || github.event.merge_group.base_sha }}" 
+# API_DIFF_OLD_COMMIT: commit before the API changes to compare with. If not provided, script will fail with "unbound variable" error
+# API_DIFF_NEW_COMMIT: commit with the new API changes. If not provided, script will fail with "unbound variable" error
 # TARGET_BREAKING_CHANGES_FILE - file to save breaking changes
+TARGET_BREAKING_CHANGES_FILE=${TARGET_BREAKING_CHANGES_FILE:-""}
 script_path=$(dirname "$0")
 
```

Comment:
We should check if values are not present - error or offer sensible defaults?

Reply:
Nit. can be done later.

Reply:
script will fail if vars are not passed, they're required